frontend/src/libs/components/image-selector/libs/enums/default-settings.enum.ts

```diff
@@ -0,0 +1,10 @@
+import { type OutputFormat, type ViewMode } from '../types/types.js';
+
+const DefaultSettings = {
+  VIEW_MODE: 'square' satisfies ViewMode as ViewMode,
+  INITIAL_ZOOM: 0.8,
+  MODAL_SIZE: 400,
+  OUTPUT_FORMAT: 'png' satisfies OutputFormat,
+} as const;
```

Comment:
I don't this that is actually an enum, it's more bout default configuration, so move it out from enums, and remove as const
and add the return type to this object

Reply:
also I'm thinking to add our own enums OutputFormat and ViewMode to write needed values here, and to avoid writing 'satisfies ViewMode as ViewMode'

Reply:
Yeah, did it